crates/parser/src/abi.rs

```diff
@@ -10,9 +10,8 @@ pub struct WasmSignature {
     pub params: Vec<WasmType>,
     /// The WebAssembly results of this function.
     pub results: Vec<WasmType>,
-    /// The raw types, if needed, returned through return pointer located in
-    /// `params`.
-    pub retptr: Option<Vec<WasmType>>,
+    /// TODO
```

Comment:
I imagine this TODO (and the one below for `return_pointer`) will be short lived, but can we add an explanation just in case?